New dev videos about the experimental <selectmenu> element and devtools what's new (#1965)
MicrosoftDocs/edge-developer: microsoft-edge/dev-videos/index.md

-->

<!-- ====================================================================== -->
## Web Platform - Fully style the drop-down part of a `<select>` with the new `<selectmenu>` element

_Published on 31 May 2022._

[![Thumbnail image for the selectmenu video](./images/selectmenu.png)](https://www.youtube.com/watch?v=Ts7jvRyQACY)

> Styling `<select>` elements has been difficult for a long time, but the experimental `<selectmenu>` element promises to overcome the remaining limitations in the future and let web developers style all parts of the element.

Reviewer: ```suggestion
Styling `<select>` elements has been challenging. The experimental `<selectmenu>` element promises to overcome the remaining limitations by enabling web developers to style all parts of the element.
```
